src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -372,6 +376,8 @@ private AIState lookForNeededItems()
 
     /**
      * Updates the itemsCurrentlyNeeded with current values.
+     * 
+     * This function seems bugged, as it gives itemsCurrentlyNeeded = isEmpty 
```

Comment:
after further inventigation I found the place where no items get pulled in anymore is
`itemsNeeded.forEach(job::addItemNeeded);`

In the attached log you can see right after the builder did one action, the above will not fill 
job anymore
[latest.txt](https://github.com/Minecolonies/minecolonies/files/716454/latest.txt)